src/fetching/postCampaign.ts

```diff
@@ -0,0 +1,17 @@
+import { defaultFetch } from '.';
+import { ZetkinCampaign } from '../types/zetkin';
+
+export default function postCampaign(orgId : string, fetch = defaultFetch) {
+    return async (action: Record<string, unknown>):Promise<ZetkinCampaign> => {
```

Comment:
As I just learned from @djbusstop, the type of the first argument could be `Partial<Campaign>` which means that some only fields need to be available. Although I guess if one wants to be really strict, there should be a special type for the fields that are actually required. So lets leave it like this.

The name of the parameter should probably not be `action` though. Copy-paste mistake?

Reply:
Cool! Did not know